### DISCUSSION TITLE
error while referencing the table(foreign key array)

### DISCUSSION THREAD

**Discussion by himeshp** [supabase/supabase#1909] [Questions] [answered]
<img width="891" alt="Screenshot 2021-06-10 at 3 40 31 PM" src="https://user-images.githubusercontent.com/5878422/121507482-4ae1f600-ca02-11eb-89aa-4d49f72f17f3.png">

i have 2 tables,
news and tags
and I am trying to reference the tag name in news table on field which is defined as array of text

**Comment by steve-chavez** [comment, marked as answer]
Hey @himeshp,

Foreign key arrays are not supported, you'll need to create an additional junction table. For more details, see this previous discussion: https://github.com/supabase/supabase/discussions/535

**Reply by himeshp** [reply]
Thanks for the update..!!
